### PR TITLE
Measure in-cluster network latency with more granularity.

### PR DESCRIPTION
**Pull request by oxddr**
Also:
 * count errors
 * fix golint issues around naming.

ref https://github.com/kubernetes/perf-tests/issues/597
/assign @wojtek-t

### CODE REVIEW

File: probes/pkg/ping/client/client.go [hunk 1]
```diff
@@ -21,50 +21,91 @@ import (
...
 
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/perf-tests/probes/pkg/common"
+
 	"k8s.io/klog"
-	"k8s.io/perf-tests/probes/pkg/metrics"
 )
 
 var (
+	// inClusterNetworkLatency implements the In-Cluster Network Programming SLI, see
```
Comment (wojtek-t) [inline probes/pkg/ping/client/client.go]: can we move it to metrics.go in this directory?

Comment (oxddr) [reply probes/pkg/ping/client/client.go]: Sure, but why?

Comment (wojtek-t) [reply probes/pkg/ping/client/client.go]: We're generally trying to define metrics in metrics.go files in all kubernetes - it's just easier to find metrics then.

Comment (oxddr) [reply probes/pkg/ping/client/client.go]: sgtm

Is is documented/defined/written somewhere?

Comment (wojtek-t) [reply probes/pkg/ping/client/client.go]: not that I'm aware of

File: probes/pkg/dns/metrics.go [hunk 2]
```diff
@@ -14,24 +14,36 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package metrics
+package dns
```
Comment (wojtek-t) [inline probes/pkg/dns/metrics.go]: Hmm - github suggests you move pkg/metrics/metrics to this one. Is that true? Or is it a copy?

Comment (wojtek-t) [reply probes/pkg/dns/metrics.go]: OK - never mind, I see there are two files now.

**Pull request merged**